### PR TITLE
[enterprise-4.12] Manual CP OBSDOCS-848 Logging 5.6.17 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
This is a manual cherry-pick of https://github.com/openshift/openshift-docs/pull/73273

Fix version: 4.12

Doc Preview: https://73854--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/logging_release_notes/logging-5-6-release-notes#logging-release-notes-5-6-17_logging-5-6-release-notes

cherry picked from: 1797950cc1d976eac3d3843a07f7816ac27a4497

### CODE REVIEW

File: modules/logging-release-notes-5-6-17.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-17_{context}"]
+= Logging 5.6.17
+This release includes link:https://access.redhat.com/errata/RHSA-2024:1507[OpenShift Logging Bug Fix 5.6.17]
```
Comment (bergerhoffer) [inline modules/logging-release-notes-5-6-17.adoc:5]: In your original PR, I missed that this is missing a period. I'll merge it for now, but if you do any updates to this file in the future, would you mind adding it in across versions?

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging!

File: modules/logging-release-notes-5-6-17.adoc (hunk 1 above)
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-6-17.adoc:5]: Sure thing! I have some grooming to do for Logging z-stream releases anyway. Will do it as a part of that task.

**Pull request merged**